### PR TITLE
MarekJasinski.FreeCommanderXE version 2023.0.0.880

### PR DESCRIPTION
**Pull request by borque**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

### CODE REVIEW

**Comment by borque** [comment]
Closes #93309

**Review by borque — COMMENTED** [review]
Closes #93603

File: manifests/m/MarekJasinski/FreeCommanderXE/2023.0.0.880/MarekJasinski.FreeCommanderXE.locale.en.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+PublisherUrl: https://freecommander.com/en/summary/
+PublisherSupportUrl: https://freecommander.com/fchelpxe/en/FreeCommander.html
+PrivacyUrl: https://freecommander.com/en/privacy-policy/
+Author: Marek Jasinski
+PackageName: FreeCommander XE
+PackageUrl: https://freecommander.com/en/summary/
+License: Proprietary
+LicenseUrl: https://freecommander.com/en/license/
+Copyright: Copyright © 2004-2023 Marek Jasinski. All rights reserved
+ShortDescription: FreeCommander XE is a Norton-Commander-clone-style file manager.
```
Comment (stephengillie) [inline manifests/m/MarekJasinski/FreeCommanderXE/2023.0.0.880/MarekJasinski.FreeCommanderXE.locale.en.yaml:17]:
```suggestion
ShortDescription: FreeCommander XE is an easy-to-use alternative to the standard windows file manager.
```

File: manifests/m/MarekJasinski/FreeCommanderXE/2023.0.0.880/MarekJasinski.FreeCommanderXE.locale.de.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+PublisherUrl: https://freecommander.com/de/ubersicht/
+PublisherSupportUrl: https://freecommander.com/fchelpxe/de/FreeCommander.html
+PrivacyUrl: https://freecommander.com/de/datenschutz/
+Author: Marek Jasinski
+PackageName: FreeCommander XE
+PackageUrl: https://freecommander.com/de/ubersicht/
+License: Proprietary
+LicenseUrl: https://freecommander.com/de/lizenz/
+Copyright: Copyright © 2004-2023 Marek Jasinski. All rights reserved
+ShortDescription: Total Commander ist ein Filemanager im Stil des Norton Commanders.
```
Comment (stephengillie) [inline manifests/m/MarekJasinski/FreeCommanderXE/2023.0.0.880/MarekJasinski.FreeCommanderXE.locale.de.yaml:17]:
```suggestion
ShortDescription: FreeCommander XE ist eine leicht bedienbare Alternative zum Standard-Dateimanager.
```

Comment (stephengillie) [reply manifests/m/MarekJasinski/FreeCommanderXE/2023.0.0.880/MarekJasinski.FreeCommanderXE.locale.de.yaml:17]: Hi @borque,

Same situation as above, and also the product name is slightly different. I'm not very familiar with German, only working with internet translations, so feel free to update this if needed.

**Comment by borque** [comment]
Hello @stephengillie !
This change is no problem. As you have noticed I reused the description texts from Total Commanders manifest, which I also authored (and which also caused a Policy 1.2-Error).
Indeed "Norton-Commander-Clone" is a kind of defined term for all filemanagers imitating the two-window-style of Norton-Commander from the DOS-era  in Germany.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**